### PR TITLE
refactor: replace serving claim with notify claim (CMC-1205)

### PR DESCRIPTION
**Pull request by devmarcy**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/CMC-1205

### Change description ###
Changed eligibility screen which now says notify claim as opposed to serving the claim
On confirmation screen made content change page to say that 'claimant has until DATE to notify the defendant of the claim and claim details'
Changed steps.js to reflect the change made on the confirmation screen

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/unspec/handler/callback/user/CreateClaimCallbackHandler.java [hunk 1]
```diff
@@ -192,8 +188,6 @@ private SubmittedCallbackResponse buildConfirmation(CallbackParams callbackParam
 
         String body = format(
             CONFIRMATION_SUMMARY,
-            format("/cases/case-details/%s#CaseDocuments", caseData.getCcdCaseReference()),
-            claimIssueConfiguration.getResponsePackLink(),
             formattedServiceDeadline
```
Comment (ckalista) [inline src/main/java/uk/gov/hmcts/reform/unspec/handler/callback/user/CreateClaimCallbackHandler.java:192]: We will work out what the actual deadline in another story (so we are good here)

**Review by ckalista — DISMISSED** [review]

File: src/main/java/uk/gov/hmcts/reform/unspec/handler/callback/user/CreateClaimCallbackHandler.java [hunk 2]
```diff
@@ -192,8 +188,6 @@ private SubmittedCallbackResponse buildConfirmation(CallbackParams callbackParam
 
         String body = format(
             CONFIRMATION_SUMMARY,
-            format("/cases/case-details/%s#CaseDocuments", caseData.getCcdCaseReference()),
-            claimIssueConfiguration.getResponsePackLink(),
```
Comment (pd-dudkiewicz) [inline src/main/java/uk/gov/hmcts/reform/unspec/handler/callback/user/CreateClaimCallbackHandler.java:198]: `ClaimIssueConfiguration` class is not used anymore, so could be removed (also remove `unspecified.response-pack-url` from config and callback handler test).

Comment (devmarcy) [reply src/main/java/uk/gov/hmcts/reform/unspec/handler/callback/user/CreateClaimCallbackHandler.java:198]: Should I remove just usages or should I just delete the class as well?

**Review by swalker125 — COMMENTED** [review]
So you can actually delete a little more with this PR. Can remove response-pack-url in application.yml, also can delete ClaimIssueConfiguration class and any tests associated.

File: src/main/java/uk/gov/hmcts/reform/unspec/handler/callback/user/CreateClaimCallbackHandler.java (hunk 2 above)
Comment (pd-dudkiewicz) [reply src/main/java/uk/gov/hmcts/reform/unspec/handler/callback/user/CreateClaimCallbackHandler.java:198]: This was the only place where configuration class was used, so let's remove it

**Review by pd-dudkiewicz — DISMISSED** [review]

**Review by dharmendrak — DISMISSED** [review]

**Review by douglasrice — CHANGES_REQUESTED** [review]
Please revert the bullet point 1
From: issuing a claim for an unspecified (unfixed) amount of money
To: issuing a claim for damages

**Pull request merged**